Shared/sdk/CVector.h

```diff
@@ -261,4 +267,6 @@ class CVector
     }
 
     bool operator!=(const CVector& param) const noexcept { return !(*this == param); }
+
+    float operator[](size_t i) const noexcept { return ((float*)this)[i]; }
```

Comment:
```suggestion
    float operator[](size_t i) const { return ((float*)this)[i]; }
```

Reply:
All other operators are noexcept too, why change it?

Reply:
Because you can access an element out-of-range, and even the STL does not use `noexcept` for `operator[]` in its classes.